### PR TITLE
[SYCL] Replace using of ONEAPI/INTEL namespaces with ext::oneapi/intel

### PR DESCRIPTION
**Pull request by maximdimakov**
ONEAPI/INTEL namespaces are deprecated in SYCL 2020, they was changed to ext::oneapi/intel namespaces
Namespaces are removed in : https://github.com/intel/llvm/pull/4488

### CODE REVIEW

**Review by dm-vodopyanov — APPROVED** [review]
All changes LGTM. Patch has a lot of approvers, @vladimirlaz, can it be merged with the less amount of approvals as the changes are the same in the whole files?

**Comment by vladimirlaz** [comment]
> All changes LGTM. Patch has a lot of approvers, @vladimirlaz, can it be merged with the less amount of approvals as the changes are the same in the whole files?

We can merge the patch without the full list of approvals, but it is better to collect several.

**Review by romanovvlad — APPROVED** [review]

**Review by vladimirlaz — APPROVED** [review]

**Review by AGindinson — APPROVED** [review]

**Pull request merged**
